src/components/views/elements/AppTile.js

```diff
@@ -274,6 +274,11 @@ export default class AppTile extends React.Component {
     }
 
     _canUserModify() {
+        // User widgets should always be modifiable by their creator
+        if (this.props.userWidget && MatrixClientPeg.get().credentials.userId === this.props.creatorUserId) {
```

Comment:
I've recently been pointed to get the MatrixClient instance over React Contexts, probably worth pointing out here also